tests/workflows/basic_code_execution_node/try_workflow.py

```diff
@@ -8,7 +8,7 @@
 
 @TryNode.wrap()
 class SimpleCodeExecutionNode(CodeExecutionNode[BaseState, int]):
-    filepath = "/".join(base_module + ["tests", "code.py"])
+    filepath = "tests/code.py"
```

Comment:
```suggestion
    filepath = "./tests/code.py"
```

Can we quickly confirm that ^ works too (here and below)

Reply:
Yeah that works, I will just update the PR with those paths to be more inline with our `./script.py` convention